lib/ui/collections/album/column_item.dart

```diff
@@ -9,10 +9,10 @@ import 'package:photos/ui/viewer/file/no_thumbnail_widget.dart';
 import 'package:photos/ui/viewer/file/thumbnail_widget.dart';
 
 ///https://www.figma.com/file/SYtMyLBs5SAOkTbfMMzhqt/ente-Visual-Design?node-id=7480%3A33462&t=H5AvR79OYDnB9ekw-4
-class AlbumListItemWidget extends StatelessWidget {
+class AlbumColumnItemWidget extends StatelessWidget {
```

Comment:
Let's make sure we change the name of the component in figma too when we are changing it in code

Reply:
Let's rename this particular on Figma? We list the albums as either list or row (multi grid row)